src/excmds.ts

```diff
@@ -894,8 +894,7 @@ export async function curJumps() {
     }
     const page = getJumpPageId()
     ensure(jumps, page, {})
-    const dummy = new UIEvent("scroll")
-    ensure(jumps[page], "list", [{ x: dummy.pageX, y: dummy.pageY }])
+    ensure(jumps[page], "list", [{ x: 0, y: 0 }])
```

Comment:
I think we need to use `window.pageX` and `window.pageY` here - when users navigate to a page with an anchor (e.g. `http://w.x/y#z`) they won't be at the page's top.